pages/components/Alert/alert.css

```diff
@@ -0,0 +1,113 @@
+:root {
+  --transparent-color: #efcb2d00;
+}
+
+.alert {
+  position: relative;
+  padding: 1rem 1rem;
+  margin-bottom: 1rem;
+  border: 1px solid transparent;
```

Comment:
What's the need of `border: 1px solid transparent;`
as it does not make any difference.

Reply:
It is used to provide a hairline gap of 1px in the container, Improves the UI